common-util/functions/requests.ts

```diff
@@ -4,7 +4,11 @@ const ESTIMATED_GAS_LIMIT = 2_000_000;
 /**
  * function to estimate gas limit
  */
-export const getEstimatedGasLimit = async (fn, account, value) => {
+export const getEstimatedGasLimit = async (
+  fn: { estimateGas: (arg0: { from: string; value: number | undefined }) => number },
```

Comment:
can you try using types from monorepo? for consistency https://github.com/valory-xyz/autonolas-frontend-mono/blob/main/libs/util-functions/src/lib/requests.ts#L9